examples/run_demo_ursula_fleet.py

```diff
@@ -0,0 +1,73 @@
+"""
+This file is part of nucypher.
+nucypher is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+nucypher is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
+"""
+
+# WARNING This is not a mining script!
+# you will not perform any re-encryptions, and you will not get paid.
+# It might be (but might not be) useful for determining whether you have
+# the proper dependencies and configuration to run an actual mining node.
+import os
+from contextlib import suppress
+from functools import partial
+
+from twisted.internet import reactor
+
+from nucypher.characters.lawful import Ursula
+
+FLEET_POPULATION = 12
+DEMO_NODE_STARTING_PORT = 11500
+
+ursula_maker = partial(Ursula, rest_host='127.0.0.1',
+                       federated_only=True,
+                       domains=["finnegans/heartbeat/whatever"]
+                       )
+
+
+def spin_up_federated_ursulas(quantity: int = FLEET_POPULATION):
```

Comment:
Not using docker - so a full development installation is needed, correct?

Reply:
:thinking: is that so?  I can't immediately think of a requirement that's only satisfied in the dev installation.  

It doesn't use pytest, doesn't require appdirs, doesn't use prometheus... 

But it's really only meant for development anyway, so it won't bother me if it does.